contract/models/contract.py

```diff
@@ -217,12 +212,61 @@ def action_contract_send(self):
             'context': ctx,
         }
 
+    @api.model
+    def _get_recurring_create_invoice_domain(self, contract=False):
```

Comment:
I was thinking of `_get_contracts_to_invoice_domain(self, date_ref=None, contracts=None)`